.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: Build Docker image
+
+on:
+  pull_request:
```

Comment:
Do you know if this gets called every time that somebody updates a pull request? Or is it just on pull request creation? Because if it only runs when the pull request is created, then that would create a loophole.

Reply:
This will run every time someone updates a pr. For example, you can see the list of times workflow ran as I was updating this PR here https://github.com/conda-incubator/conda-store-ui/actions

Reply:
Great! Marking this question as resolved.